model/src/main/kotlin/config/CopyrightBlacklist.kt

```diff
@@ -0,0 +1,8 @@
+package com.here.ort.model.config
+
+import java.util.*
+
+data class CopyrightBlacklist(val items: Set<String> = setOf()) {
+    constructor(vararg items: String) : this(items.toSortedSet())
```

Comment:
`toSet()` should be sufficient since the type of `items` in the primary constructor is also not a sorted set.